lib/prism.js

```diff
@@ -97,7 +98,9 @@ function PrismUtil(str, options = {}) {
 
   if (tab) str = replaceTabs(str, tab);
 
-  const startTag = `<pre class="${preTagClassArr.join(' ')}"${preTagAttr}><code class="language-${language}">`;
+  const figCaption = caption ? `<figcaption>${caption}</figcaption>` : '';
```

Comment:
https://mdn.io/figcaption

`<figcaption>` is only valid in `<figure>` element.